addons/loyalty/data/loyalty_demo.xml

```diff
@@ -72,4 +72,30 @@
         <field name="mail_template_id" ref="loyalty.mail_template_loyalty_card"/>
         <field name="program_id" ref="loyalty.10_percent_coupon"/>
     </record>
+    <!-- second product at 50% -->
+    <record id="second_product_50_percent" model="loyalty.program">
+        <field name="name">50% off on second cheapest chair</field>
+        <field name="program_type">promotion</field>
+        <field name="applies_on">current</field>
+        <field name="trigger">auto</field>
+        <field name="portal_visible">False</field>
+        <field name="portal_point_name">Promo points</field>
+    </record>
+
+    <record id="second_product_50_percent_rule" model="loyalty.rule">
+        <field name="minimum_qty">2</field>
+        <field name="reward_point_mode">unit</field>
+        <field name="reward_point_amount">1</field>
+        <field name="program_id" ref="loyalty.second_product_50_percent"/>
+    </record>
+
+    <record id="second_product_50_percent_reward" model="loyalty.reward">
+        <field name="reward_type">discount</field>
+        <field name="discount">50</field>
+        <field name="discount_mode">percent</field>
+        <field name="discount_applicability">cheapest</field>
+        <field name="discount_product_domain">[("name", "ilike", "Chair")]</field>
```

Comment:
Note to self: could move the chair category from POS instead to not have the chair floor protection included. Worth it or don't care? We never use product domain in demo data, might be the occasion :shrug:

Reply:
Was decided to add 4 specific chair products instead when moving this commit in another PR, so problem solved